app/config/FrontendAppConfig.scala

```diff
@@ -311,6 +311,7 @@ class FrontendAppConfig @Inject()(val servicesConfig: ServicesConfig, runModeCon
   lazy val businessSignInLink = "https://www.gov.uk/guidance/sign-in-to-your-hmrc-business-tax-account"
   lazy val fhddsRegisteredBusinessesListUrl = "https://www.gov.uk/government/publications/fulfilment-house-due-diligence-scheme-registered-businesses-list"
   lazy val regime: String = "VATC"
+  lazy val businessVerificationStatus = true
```

Comment:
not really necessary to set up booleans like that in appConfig, could just pass them in normally where needed

Reply:
I was thinking this would help if need switching the default value by only changing the config that will reflect everywhere. But I am also fine with passing the value directly.